ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+# Backup collections
+
+Backup collections provide an advanced backup solution for {{ ydb-short-name }} that organizes full and incremental backups into managed collections. This approach is designed for production workloads requiring efficient disaster recovery and point-in-time recovery capabilities.
+
+## What are backup collections? {#what-are-backup-collections}
+
+A backup collection is a named set of coordinated backups for selected database objects. Collections organize related backups and ensure they can be restored together consistently, providing:
+
+- **Organization** — related backups are grouped into logical collections.
+- **Recovery flexibility** — enables recovery to any point in time when one of the backups in the collection was made.
+
+{% note info %}
+
+Currently only row-oriented tables are supported.
+
+{% endnote %}
+
+## Core concepts {#core-concepts}
+
+### Full backup {#full-backup}
+
+A snapshot at a specific point in time. Serves as the foundation for subsequent incremental backups and contains all data necessary for independent restoration.
+
+### Incremental backup {#incremental-backup}
+
+Captures only changes (inserts, updates, deletes) since the previous backup in the chain. Significantly smaller than full backups with low change frequency.
+
+### Backup chain {#backup-chain}
+
+An ordered sequence of backups starting with a full backup followed by zero or more incremental backups. Each incremental backup depends on all previous backups in the chain.
+
+## Backup workflow {#backup-workflow}
```

Comment:
Add an intro paragraph with high-level explanation of what's going on in the following diagram